includes/wp-admin/class-settings.php

```diff
@@ -272,6 +242,36 @@ public static function register_settings() {
 			)
 		);
 
+		\register_setting(
+			'activitypub_blog',
+			'activitypub_mailer_new_dm',
+			array(
+				'type'        => 'boolean',
+				'description' => 'Send a notification when someone sends a user of the blog a direct message.',
+				'default'     => false,
```

Comment:
this is really weird... we set the default to `false` and add an (user_)option instead. Let's fix that as part of this PR. we should not rely on that after the re-write.

Reply:
Updated in [1b93797](https://github.com/Automattic/wordpress-activitypub/pull/1586/commits/1b937971ab5c7fbc3d193b3db474ba53144e1736)